src/saltext/vault/states/vault_pki.py

```diff
@@ -0,0 +1,502 @@
+
+    try:
+        # check file.managed changes early to avoid using unnecessary resources
+        file_managed_test = _file_managed(name, test=True, replace=False, **file_args)
+        if file_managed_test["result"] is False:
+            ret["result"] = False
+            ret["comment"] = "Problem while testing file.managed changes, see its output"
+            _add_sub_state_run(ret, file_managed_test)
+            return ret
+
+        if "is not present and is not set for creation" in file_managed_test["comment"]:
+            _add_sub_state_run(ret, file_managed_test)
+            return ret
+
+        # handle follow_symlinks
+        if __salt__["file.is_link"](name):
+            if file_args.get("follow_symlinks", True):
+                name = os.path.realpath(name)
+            else:
+                # workaround https://github.com/saltstack/salt/issues/31802
+                __salt__["file.remove"](name)
+                changes["replaced"] = True
+
+        replace = False
+        file_exists = __salt__["file.file_exists"](name)
+
+        if issuer_ref is None:
+            issuer_ref = __salt__["vault_pki.read_role"](role_name, mount=mount)["issuer_ref"]
+
+        issuer_info = __salt__["vault_pki.read_issuer"](issuer_ref, mount=mount)
+
+        if append_ca_chain:
+            ca_chain = [x509util.load_cert(x) for x in issuer_info["ca_chain"]]
+
+        if file_exists:
+            if reissue:
+                # No need to make any checks, just replace the cert
+                changes["replaced"] = True
+            else:
+
+                changes = check_cert_for_changes(
+                    current=name,
+                    append_chain=ca_chain,
+                    common_name=common_name,
+                    encoding=encoding,
+                    issuer=issuer_info["certificate"],
+                    private_key=private_key,
+                    private_key_passphrase=private_key_passphrase,
+                    common_name_only=not sign_verbatim,
+                    expire_tolerance=ttl_remaining,
+                    **cert_args,
+                )
+
+        else:
+            changes["created"] = True
+
+        if not changes and file_managed_test["result"] and not file_managed_test["changes"]:
+            _add_sub_state_run(ret, file_managed_test)
+            return ret
+
+        ret["changes"] = changes
+        if changes and file_exists:
+            verb = "reissue"
+
+        if __opts__["test"]:
+            ret["result"] = None if changes else True
+            ret["comment"] = (
+                f"The certificate would have been {verb}d" if changes else ret["comment"]
+            )
+            _add_sub_state_run(ret, file_managed_test)
+            return ret
+
+        if changes:
+            if not set(changes) - {
+                "ca_chain",
+                "encoding",
+            }:
+                verb = "recreat"
```

Comment:
Now it's missing the `e` ^^

```suggestion
                verb = "recreate"
```

It's printed as `{verb}d`.